code/model/editableformfields/EditableTextField.php

```diff
@@ -14,6 +14,8 @@ class EditableTextField extends EditableFormField
 
     private static $plural_name = 'Text Fields';
 
+    protected $jsEventHandler = 'keyup';
```

Comment:
I've seen this a few times now, does it need to be defined (with its doc block) in `EditableFormField` instead?